etc/HIReplayOfflineConfiguration.py

```diff
@@ -1318,414 +1465,296 @@
+               write_dqm=True,
+               alca_producers=["TkAlMinBias"],
+               dqm_sequences=["@common"],
+               scenario=hiTestppScenario)
 
-datasets = [ "HIHeavyFlavor", "HIHighMultiplicityETTAsym" ]
+DATASETS = ["HIOnlineMonitor", "HITrackerNZS"]
 
-for dataset in datasets:
+for dataset in DATASETS:
     addDataset(tier0Config, dataset,
-               write_miniaod = False,
-               do_reco = True,
-               write_dqm = True,
-               reco_split = hiRecoSplitting,
-               dqm_sequences = [ "@common" ],
-               scenario = hiTestppScenario)
+               do_reco=False,
+               raw_to_disk=True,
+               scenario=hiTestppScenario)
 
-datasets = [ "HIMinimumBiasReducedFormat0", "HILowMultiplicityReducedFormat", "HILowMultiplicity" ]
+DATASETS = ["HIEmptyBX"]
 
-for dataset in datasets:
+for dataset in DATASETS:
     addDataset(tier0Config, dataset,
-               write_miniaod = False,
-               do_reco = True,
-               raw_to_disk = True,
-               write_dqm = True,
-               dqm_sequences = [ "@common" ],
-               scenario = hiTestppScenario)
+               do_reco=True,
+               write_dqm=True,
+               dqm_sequences=["@common"],
+               scenario=hiTestppScenario)
 
-# CMS VdM scan PDs
-datasets = [ "HICentralityVetoReducedFormat0", "HICentralityVetoReducedFormat1", "HICentralityVetoReducedFormat2",
-             "HICentralityVetoReducedFormat3", "HICentralityVetoReducedFormat4", "HICentralityVetoReducedFormat5",
-             "HICentralityVetoReducedFormat6", "HICentralityVetoReducedFormat7", "HICentralityVetoReducedFormat8",
-             "HICentralityVetoReducedFormat9", "HICentralityVetoReducedFormat10", "HICentralityVetoReducedFormat11" ]
+DATASETS = ["HITestRaw0", "HITestRaw1", "HITestRaw2", "HITestRaw3", "HITestRaw4", "HITestRaw5",
+            "HITestRaw6", "HITestRaw7", "HITestRaw8", "HITestRaw9", "HITestRaw10", "HITestRaw11",
+            "HITestRaw12", "HITestRaw13", "HITestRaw14", "HITestRaw15", "HITestRaw16", "HITestRaw17",
+            "HITestRaw18", "HITestRaw19", "HITestRaw20", "HITestRaw21", "HITestRaw22", "HITestRaw23"]
 
-for dataset in datasets:
+for dataset in DATASETS:
     addDataset(tier0Config, dataset,
-               write_miniaod = False,
-               do_reco = True,
-               write_dqm = True,
-               dqm_sequences = [ "@common" ],
-               scenario = hiTestppScenario)
+               write_miniaod=False,
+               do_reco=True,
+               raw_to_disk=True,
+               write_dqm=True,
+               alca_producers=["SiStripCalZeroBias", "SiStripCalMinBias","TkAlMinBias",
+                               "HcalCalIsolatedBunchSelector", "HcalCalIterativePhiSym","HcalCalMinBias",
+                               "TkAlJpsiMuMu", "TkAlUpsilonMuMu","TkAlZMuMu","TkAlMuonIsolated"],
+               dqm_sequences=["@commonSiStripZeroBias"],
```

Comment:
@germanfgv please note that in [1c2f703](https://github.com/dmwm/T0/pull/4778/commits/1c2f703d2e8a1767fbdbec287737e9b828815421) (last commit of my replay) I added some `dqm_sequences` here: if the replay there is successful we should add them here as well.

Reply:
done